web/src/app/alerts/components/AlertDetails.tsx

```diff
@@ -28,22 +29,50 @@ import {
   SlackChannelLink,
   UserLink,
 } from '../../links'
-import { styles } from '../../styles/materialStyles'
+import { styles as globalStyles } from '../../styles/materialStyles'
 import Markdown from '../../util/Markdown'
 import AlertDetailLogs from '../AlertDetailLogs'
 import AppLink from '../../util/AppLink'
 import { useIsWidthDown } from '../../util/useWidth'
 import CardActions from '../../details/CardActions'
 import Notices from '../../details/Notices'
+import {
+  Alert,
+  Target,
+  EscalationPolicyStep,
+  Notice,
+  AlertStatus,
+} from '../../../schema'
+
+interface AlertDetailsProps {
+  data: Alert
+}
 
-const useStyles = makeStyles((theme) => {
-  return {
-    ...styles(theme),
-    epHeader: {
-      paddingBottom: 8,
-    },
-  }
-})
+interface MenuOption {
+  icon: ReactElement
+  label: string
+  handleOnClick: () => void
+}
+
+interface EscalationPolicyProps {
```

Comment:
What do you think about renaming this to Info or something similar? Since it's being used mainly for our render functions instead of being passed to an EP component

```suggestion
interface EscalationPolicyInfo {
```